lib/topic.ts

```diff
@@ -3,11 +3,11 @@ import { showElementActions } from './kafka';
 import { navigateToKafkaList } from './navigation';
 
 export const navigateToKafkaTopicsList = async function (page: Page, kafkaName: string) {
-  await expect(page.getByText(kafkaName)).toHaveCount(1);
-  await page.locator('a', { hasText: kafkaName }).click();
-  await expect(await page.locator('button[aria-label="Topics"]')).toHaveCount(1);
+  await expect(page.getByText(`${kafkaName}`)).toHaveCount(1);
+  await page.locator('a', { hasText: `${kafkaName}` }).click();
+  await expect(page.locator('button[aria-label="Topics"]')).toHaveCount(1, { timeout: 10000 });
   // data-testid=pageKafka-tabTopics
-  await page.locator('button[aria-label="Topics"]').click();
+  await page.locator('button[aria-label="Topics"]').click({ timeout: 10000 });
```

Comment:
Why do we decrease it from 30 to 10s? Or is the default value different?

Reply:
by allowing 10 seconds on those steps that were the ones to repeatedly fail we are allowing time for the buttons to load.

Reply:
Oh ye, we don't have timeout for clicking. Anyway line 8 waits for the button with timeout 30s so when PW click on it, it should be here for sure. I assume it doesn't work for you, why?

Reply:
well, the traces when it failed showed the Button was not there, and therefore could not be clicked. With this Timeout I got 2 sucessful executions in a row, and it has already passed PR Pipeline

Reply:
I think we should understand why the current approach doesn't work. I am not against adding the timeout, but line 8 should have 30s now, you decrease it to 10s and you add an additional timeout to line 10 that should pass immediately because of line 8.

Reply:
Can I resolve this conversation?

Reply:
which kind of problems? The code should behave as expected especially when we wait for element X for 30s, in case that expect does not fail it means the X is available. Then we click on X and this fails - WHY?

Reply:
the failures that lead to retrials. I am not really sure the test behaves like that but forcing the assertion to happen for these specified 10 seconds

Reply:
Sorry to be late to this thread. The conversation is confusing. 

I agree that we want to avoid "magic numbers" - if the default timeout is 30 seconds, are you saying that reducing this specific timeout to 10 seconds is resulting in successful tests executions?

Reply:
No, I am not sure that at that step the timeout of 30 seconds was happening

Reply:
This does not seem right @asanzgom, could you please try a negative test to verify that the PW config for 30 sec expect timeout is truly not applied and therefore there is a bug in Playwright? And upload a trace here that would be showing `expect timeout reached 30 sec` or `expect timeout reached 5 sec` or whatever?

Reply:
@mmusil good idea, thanks

Reply:
And - on line 8 - we have a default expect timeout of 30, and the change reduces this to 10?